src/store/ChartSettingStore.ts

```diff
@@ -117,9 +121,7 @@ export default class ChartSettingStore {
         if (isHighestLowestMarkerEnabled !== undefined) {
             this.toggleHighestLowestMarker(isHighestLowestMarkerEnabled);
         }
-        if (whitespace !== undefined) {
-            this.setWhiteSpace(whitespace);
-        }
+        this.setWhiteSpace(whitespace);
```

Comment:
We need to set whitespace as `undefined` if this setting was updated (turned off) after chart initialization.